src/commands/hosting-channel-create.ts

```diff
@@ -45,11 +45,13 @@ export default new Command("hosting:channel:create [channelId]")
             `"channelId" argument must be provided in a non-interactive environment`
           );
         }
-        channelId = await promptOnce({
-          type: "input",
-          message: "Please provide a URL-friendly name for the channel:",
-          validate: (s) => s, // Prevents an empty string from being submitted!
-        });
+        channelId = await promptOnce(
+          {
+            type: "input",
+            message: "Please provide a URL-friendly name for the channel:",
+            validate: (s) => s.length > 0,
```

Comment:
```suggestion
            validate: (s) => s.length,
```

Reply:
A suggestion

Reply:
Letting this be, because it gives me a lint error in vscode otherwise.